xsl/xccdf-report-oval-details.xsl

```diff
@@ -128,11 +132,110 @@ Authors:
+            <xsl:variable name='comment' select='$object_info[1]/@comment'/>
+            <xsl:if test="$object_info">
+                <p>The tested object <strong>
+                <xsl:if test='$comment'>
+                    <xsl:attribute name='title'><xsl:value-of select='$object_info[1]/@comment'/></xsl:attribute>
+                </xsl:if>
+                <xsl:value-of select='$object_id'/></strong> of type
+                <strong><xsl:value-of select='local-name($object_info)'/></strong>
+                could not be found.</p>
+                <p>Object details:</p>
+                <table class="table table-striped table-bordered">
+                    <thead>
+                        <xsl:apply-templates mode='item-head' select='$object_info[1]'/>
+                    </thead>
+                    <tbody>
+                        <tr>
+                        <xsl:variable name='variable_id' select='$object_info/*/@var_ref'/>
+                        <xsl:if test='$variable_id'>
+                            <td>
+                            <xsl:choose>
+                            <xsl:when test='count(ovalres:tested_variable)>1'>
+                                <table>
+                                    <xsl:apply-templates mode='tableintable' select='ovalres:tested_variable'/>
+                                </table>
+                            </xsl:when>
+                            <xsl:when test='count(ovalres:tested_variable)=1'>
+                                    <xsl:apply-templates mode='normal' select='ovalres:tested_variable'/>
+                            </xsl:when>
+                            </xsl:choose>
+                            <xsl:apply-templates select='key("ovalsys-object",$object_id)'/>
+                            </td>
+                        </xsl:if>
+                        <xsl:apply-templates select='$object_info[1]'/>
+                        </tr>
+                    </tbody>
+                </table>
+                <xsl:if test="$state_info">
+                    <p>State details:</p>
+                    <table class="table table-striped table-bordered">
+                        <thead>
+                            <xsl:apply-templates mode='item-head' select='$state_info[1]'/>
+                        </thead>
+                        <tbody>
+                            <tr>
+                        <xsl:variable name='variable_id' select='$state_info/*/@var_ref'/>
+                        <xsl:if test='$variable_id'>
+                            <td>
+                            <xsl:apply-templates select='ovalres:tested_variable'/>
+                            <xsl:apply-templates select='key("ovalsys-object",$object_id)'/>
+                            </td>
+                        </xsl:if>
+                            <xsl:apply-templates select='$state_info[1]'/>
+                        </tr>
```

Comment:
Any reason why you aren't using @mode when applying those templates even though you use it elsewhere? Using modes is safer, it prevents templates eating up excess content.

The indentation seems off in the entire patch. And one of the hunks should be dropped because it just changes whitespace.